### ISSUE TITLE
Internal references

### ISSUE THREAD

**Issue by DCdotNerd** [JoshData/dc-code-prototype#11]
Since links to internal references to other Code sections are generated through (I assume) regular expressions, some sections get left unlinked... see, e.g. 50-1501.02(c)(5)(B)(vii).

**Comment by JoshData** [comment]
Maybe we can improve that regular expression. To copy in the citations that aren't getting linked from [50-1501.02](http://dccode.org/simple/sections/50-1501.02.html):

```
§§ 47-2862 and 47-2863
§§ 47-1805.02, 47-1807.02, and 47-1808.03 
```

Just the first section is linked in each.

@konklone @adelevie Do you think we should update the citation parser to handle multiple citations, or split it up on the DC code side?

**Comment by vdavez** [comment]
I would recommend changing `citation`. This is a use-case that goes beyond the Code itself.

I haven't looked at `citation` in a while but recall that XRegExp was eliminated as as a dependency. I don't know how hard it is to make this change. But, to my mind, it should live there.

**Comment by konklone** [comment]
Agreed, it should live in `citation`. And actually, the system does have the ability to return multiple cites per match. The trick is just isolating or adding the right regex for this.

Easiest way in this case might be to remove the `§§` requirement altogether, or to remove the `§§` when preceded by `,` or `and` or by `\d,?\s?and?\s?` or something like that. 

Alternate strategies welcome!
